force-app/main/default/classes/SOQL.cls

```diff
@@ -50,9 +53,15 @@ public with sharing class SOQL {
      * @return A list of Contact Ids.
      * Update the corresponding test class and testGetContactIdsByLastName method to cover this method
      */
-    //[public | private ] [static] data_type method_name (input parameters) {
-    //  The body of the method
-    //}
+    public static List<Id> getContactIdsByLastName () {
+        List<String> lastName = new List<String>{'Smith', 'Jones'};
+        List<Id> contactIds = new List<Id>();
+        List<Contact> conts = [SELECT Id FROM Contact WHERE LastName IN :lastName];
+        for (Contact cont : conts) {
+            contactIds.add(cont.Id);
+        }
+        return contactIds;
```

Comment:
Is it possible to make the lastNames list an argument we pass into this method?

besides that, this is a good use for a SOQL for loop or the special Map<Id, SObject> constructor which takes a List<SObject>. You can then just call keySet() and you have your Set/List of Ids.